job_templates/ci_job.xml.template

```diff
@@ -177,7 +177,7 @@ tmpd=`mktemp -d`
 mkdir $tmpd/src
 curl -sk https://raw.githubusercontent.com/ros2/ros2/master/ros2.repos -o $tmpd/ros2.repos
 vcs import $tmpd/src --input $tmpd/ros2.repos
-export CI_ARGS="$CI_ARGS --src-mounted --workspace-path $tmpd"
+export CI_ARGS="$CI_ARGS -- --src-mounted --workspace-path $tmpd"
```

Comment:
This logic could end up duplicated in another part of the code and then we might one day end up with `--ament-test-args -- -- --src-mounted --workspace-path` (which current makes ament invocations fail). I know it's not an issue for now, but I ticketed it at https://github.com/ament/ament_tools/issues/111